ibis/backends/duckdb/tests/conftest.py

```diff
@@ -59,16 +59,23 @@ def ddl_script(self) -> Iterator[str]:
                 SELECT * FROM read_parquet('{parquet_dir / f'{table}.parquet'}')
                 """
             )
-        if geospatial_supported:
+        if geospatial_supported and not SANDBOXED:
             for table in TEST_TABLES_GEO:
                 yield (
                     f"""
-                    INSTALL spatial;
-                    LOAD spatial;
                     CREATE OR REPLACE TABLE {table} AS
                     SELECT * FROM st_read('{geojson_dir / f'{table}.geojson'}')
                     """
                 )
+            yield (
```

Comment:
This makes sense, it's a slightly different than what @cpcloud  suggested but it works 🤷‍♀️